app/assets/stylesheets/header.scss

```diff
@@ -162,45 +163,46 @@ body > header {
         overflow: hidden;
         position: relative;
         max-height: 325px;
-      }
 
-      .notification_element {
-        border-bottom: 1px solid $border-grey;
-        padding: 5px;
-        min-height: 35px;
-        line-height: 18px;
-
-        > .avatar {
-          height: 35px;
-          width: 35px;
-          float: left;
-        }
+        .media.stream_element {
+          border-bottom: 1px solid $border-grey;
+          padding: 5px;
+          min-height: 35px;
+          line-height: 18px;
+          margin: 0;
 
-        .notification_message {
-          margin-left: 40px;
-          .tooltip { position: fixed; }
-        }
+          img.avatar {
+            width: 35px;
+            height: 35px;
+          }
+
+          .pull-right{
+            margin-top: 20px;
+            .aspect_membership_dropdown{ display: none; }
+          }
```

Comment:
This looks weird for notifications that only need 2 lines. In that case the eye has too much margin to the top. I'd propose the following change:
```diff
diff --git a/app/assets/stylesheets/header.scss b/app/assets/stylesheets/header.scss
index 26a6190..d49f484 100644
--- a/app/assets/stylesheets/header.scss
+++ b/app/assets/stylesheets/header.scss
@@ -176,17 +176,14 @@ body > header {
             height: 35px;
           }
 
-          .pull-right{
-            margin-top: 20px;
-            .aspect_membership_dropdown{ display: none; }
-          }
+          .pull-right > .aspect_membership_dropdown { display: none; }
+          .unread-toggle { margin: 10px; }
 
           .entypo {
             cursor: pointer;
             color: lighten($black,75%);
             font-size: 17px;
             line-height: 17px;
-            padding: 10px;
           }
 
           &.unread {
```